lib/msal-angular/README.md

```diff
@@ -44,8 +45,9 @@ At a minimum, `@azure/msal-angular` will follow the [support schedule of the mai
 
 | MSAL Angular version | MSAL support status     | Supported Angular versions |
 |----------------------|-------------------------|----------------------------|
-| MSAL Angular v2      | Active development      | 9, 10, 11, 12, 13, 14      |
-| MSAL Angular v1      | Active development      | 6, 7, 8, 9                 |
+| MSAL Angular v3      | Active development      | 15                         |
+| MSAL Angular v2      | In maintenance          | 9, 10, 11, 12, 13, 14      |
```

Comment:
Do we want to adopt "Long term support" instead of "In maintenance"? And probably mention v0 as being out of support?

Reply:
Hmmm I just used the language from our release cadence doc. Technically, from the time v3 is released, it will be in LTS.